### PR TITLE
Implemented multi-filing method (MultiFilingService->addObjectToFolder)

### PR DESCRIPTION
**Pull request by della-90**

### CODE REVIEW

File: src/Bindings/Browser/MultiFilingService.php [hunk 1]
```diff
@@ -10,22 +10,22 @@
...
-     * @param string $folderId The folder into which the object is to be filed.
-     * @param boolean $allVersions Add all versions of the object to the folder if the repository supports
-     *      version-specific filing. Defaults to <code>true</code>.
+     * @param string                      $repositoryId The identifier for the repository.
+     * @param string                      $objectId The identifier for the object.
+     * @param string                      $folderId The folder into which the object is to be filed.
+     * @param boolean                     $allVersions Add all versions of the object to the folder if the repository
+     *     supports version-specific filing. Defaults to <code>true</code>.
      * @param ExtensionDataInterface|null $extension
      */
```
Comment (sascha-egerer) [inline src/Bindings/Browser/MultiFilingService.php:31]: The `@return` doc is missing

**Comment by sascha-egerer** [comment]
Please check the formatting. It must be PSR-2 compliant

File: src/Bindings/Browser/MultiFilingService.php (hunk 1 above)
Comment (della-90) [inline src/Bindings/Browser/MultiFilingService.php:31]: Should I add it even if it does not return anything (as the Java CMIS library does)?

Comment (sascha-egerer) [inline src/Bindings/Browser/MultiFilingService.php:31]: Oh sorry! My fault! No return doc here. Please just fix the formatting. Thx!

Comment (della-90) [inline src/Bindings/Browser/MultiFilingService.php:31]: I'll fix it right now, sorry!

**Comment by della-90** [comment]
@sascha-egerer Now it should be ok :smile:

**Comment by ksjogo** [comment]
Sorry it took us so long, the original project needed to be finished.

**Pull request merged**